netbox/netbox/settings.py

```diff
@@ -340,7 +341,14 @@ def _setting(name, default=None):
     'django_prometheus.middleware.PrometheusBeforeMiddleware',
     'corsheaders.middleware.CorsMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.locale.LocaleMiddleware',
+]
+
+if ENABLE_LOCALIZATION:
```

Comment:
Minor thing. Maybe instead of conditionally inserting it, we could just remove it if ENABLE_LOCALIZATION is False:

```python
if not ENABLE_LOCALIZATION:
    MIDDLEWARE.remove("django.middleware.locale.LocaleMiddleware")
```

Seems a little cleaner then splitting the list.

Reply:
Good call, changed.